include/nil/blueprint/input_reader.hpp

```diff
@@ -372,6 +374,11 @@ namespace nil {
                 case llvm::Type::EllipticCurveTyID:
                 case llvm::Type::IntegerTyID:
                 case llvm::Type::FixedVectorTyID:{
+                    if (!value.is_object()) {
+                        std::ostringstream error;
+                        error << "Unexpected JSON element: expected object, got " << value;
```

Comment:
Can we use error level ERROR for that instead? It seems to me that the user will expect this kind of stuff to appear when boost::log::trivial::error is chosen.

Reply:
For now, it is necessary to crash with `UNREACHABLE` here, because we don't have a correct error handler in the input reader. But all these assertions should be replaced with error logs, here's an issue on it: https://github.com/NilFoundation/zkLLVM/issues/358